examples/example.ts

```diff
@@ -0,0 +1,67 @@
+const {
```

Comment:
Can you put some general comment on top, and describe what is this example about

Reply:
sorry this file was added by mistake. removed